docs/StardustDocs/topics/guides/Guides-And-Examples.md

```diff
@@ -5,22 +5,72 @@ Browse a collection of guides and examples covering key features and real-world
 </web-summary>
 
 <card-summary>
-Explore Kotlin DataFrame with user guides and real-world examples, 
+Explore Kotlin DataFrame with detailed user guides and real-world examples, 
 showcasing practical use cases and data workflows.
 </card-summary>
 
 <link-summary>
 A curated list of Kotlin DataFrame guides and examples that walk you through common operations and data analysis patterns step by step.
 </link-summary>
 
+<!--- TODO: add more guides (migration from pandas and others) and replace GH notebooks with topics --->
 
 ## Guides
 
+Explore our structured, in-depth guides to steadily improve your Kotlin DataFrame skills — step by step.
+
 * [Quickstart Guide](quickstart.md) — get started with Kotlin DataFrame in a few simple steps: 
 load data, transform it, and visualize it.
+* [Enhanced Column Selection DSL](https://blog.jetbrains.com/kotlin/2024/07/enhanced-column-selection-dsl-in-kotlin-dataframe/)
+ — explore powerful DSL for typesafe and flexible column selection in Kotlin DataFrame.
+* [40 Puzzles](https://github.com/Kotlin/dataframe/blob/master/examples/notebooks/puzzles/40%20puzzles.ipynb)
+ — inspired by [100 pandas puzzles](https://github.com/ajcr/100-pandas-puzzles).
+  An interactive guide that takes you from simple tasks to complex challenges,
+teaching you how to solve them using Kotlin DataFrame in a concise and elegant style.
+* [SQL Databases Interaction](readSqlDatabases.md)
+ — set up SQL database access and read query results efficiently into `DataFrame`.
+* [GeoDataFrame Guide](https://kotlin.github.io/kandy/geo-plotting-guide.html)
+  — explore the GeoDataFrame module that brings a convenient Kotlin DataFrame API to geospatial workflows, 
+enhanced with beautiful Kandy-Geo visualizations (*experimental*).
+* [OpenAPI Guide](https://github.com/Kotlin/dataframe/blob/master/examples/notebooks/json/KeyValueAndOpenApi.ipynb)
+ — learn how to parse and explore [OpenAPI](https://swagger.io) JSON structures using Kotlin DataFrame, 
```

Comment:
maybe mention that this is only OpenAPI 3.0.0. That will help steer users away if they use a newer version